hub/brakeman/engine.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "./../.schema/engine.json",
+  "name": "brakeman",
+  "description": "A static analysis security vulnerability scanner for Ruby on Rails applications.",
+  "url": "https://brakemanscanner.org/",
+  "areas": [],
+  "languages": [
+    "ruby"
+  ],
+  "configs": [
+    ".yml"
```

Comment:
It's too generic, @binore we should at least discuss it: whether it's even possible to detect such configs? If no, then we should not treat them sd s config.